lib/arelastic/aggregations/bucket_sort.rb

```diff
@@ -0,0 +1,35 @@
+module Arelastic
+  module Aggregations
+    class BucketSort < Arelastic::Aggregations::Aggregation
+      attr_accessor :from, :size, :sort
+
+      def initialize(name, options = {})
+        options = options.dup
+        @from = read_option! options, 'from'
+        @size = read_option! options, 'size'
+        @sort = read_option! options, 'sort'
+        super(name, options)
+      end
+
+      def as_elastic_aggregation
+        bucket_sort = {}
+        bucket_sort['sort'] = parse_sort if sort
+        bucket_sort['from'] = from if from
+        bucket_sort['size'] = size if size
+        { 'bucket_sort' => bucket_sort }
+      end
+
+      def parse_sort
```

Comment:
Should this be private? I see a test for it below, but that test should be part of the public interface?

Reply:
Made it private.